microcosm_flask/paging.py

```diff
@@ -316,11 +314,25 @@ def make_paginated_list_schema_class(cls, ns, item_schema):
         class PaginatedListSchema(Schema):
             __alias__ = "{}_list".format(ns.subject_name)
 
-            offset = fields.Integer(required=True)
-            limit = fields.Integer(required=True)
-            count = fields.Integer(required=True)
-            items = fields.List(fields.Nested(item_schema), required=True)
-            _links = fields.Raw()
+            offset = fields.Integer(
+                required=True,
+                metadata={"description": "The pagination starting offset."},
+            )
+            limit = fields.Integer(
+                required=True, metadata={"description": "The pagination limit."}
```

Comment:
Nit: comma after second arg
```suggestion
                required=True,
                metadata={"description": "The pagination limit."},
```